packages/medusa/src/services/__tests__/product-category.ts

```diff
@@ -0,0 +1,49 @@
+import { IdMap, MockRepository, MockManager } from "medusa-test-utils"
```

Comment:
suggestion(non blocking): Since it is new file, I would suggest you to create a fixture file where you create a default container with the mocks etc. You can have a look at other places like the payment provider tests. It makes the test file more readable and easy to evolve in the future if it grows. overall I think it is a better practice. it is very non blocking and I let you choose

Reply:
Thanks, always up to make tests more readable. I'll take a look and introduce this in the next endpoint.